gazebo_ros2_control/package.xml

```diff
@@ -7,6 +7,7 @@
   <maintainer email="ahcorde@osrfoundation.org">Alejandro Hernandez</maintainer>
 
   <license>BSD</license>
+  <license>Apache License 2.0</license>
```

Comment:
It might be better to add two LICENSE files (one for each package), instead of a LICENSE file in the root directory, since this package is BSD and the other is Apache 2.

Reply:
All files are Apache 2 except these two :

 - [gazebo_ros2_control_plugin.cpp](https://github.com/ros-simulation/gazebo_ros2_control/blob/79207cc7d7630a8aeced1b9f5bba985e0805e538/gazebo_ros2_control/src/gazebo_ros2_control_plugin.cpp
 -  [gazebo_ros2_control_plugin.hpp](https://github.com/ros-simulation/gazebo_ros2_control/blob/79207cc7d7630a8aeced1b9f5bba985e0805e538/gazebo_ros2_control/include/gazebo_ros2_control_plugin.hpp)

But they have changed a lot since I started to port this plugin. Does it make sense to change the License to Apache 2 too (because of the complete redesing)?

Reply:
If OSRF was the only copyright holder, I'd say it would be fine to change the license.

In this case, I can't recommend changing the license because it's not clear to me how influential the the original files were in developing this version.

I think the easiest thing to do would be to either,

1. License the entire package as BSD
2. Keep it dual licensed (list both BSD and Apache 2 in the package.xml and list both in the license file


IMO, option one is the simplest, considering an prospective user perspective. I'm open to other suggestions though.

Reply:
Added BSD 3-Clause to the LICENSE file https://github.com/ros-simulation/gazebo_ros2_control/blob/7967bb7f957636315e48635d0774a6b2de96247a/LICENSE#L204-L234

dual License in the package.xml https://github.com/ros-simulation/gazebo_ros2_control/blob/7967bb7f957636315e48635d0774a6b2de96247a/gazebo_ros2_control/package.xml#L9-L10